docs/community/ecosystem.md

```diff
@@ -0,0 +1,59 @@
+
+## Contents
+
+* [Learning MXNet and other Educational Resources](#learning-mxnet-and-other-educational-resources)
+* [New MXNet APIs](#new-mxnet-apis)
+* [Toolkits to Extend MXNet](#toolkits-to-extend-mxnet)
+* [Model Serving](#model-serving)
+* [Model Zoos](#model-zoos)
+* [MXNet as a Backend](#mxnet-as-a-backend)
+* [Contributions](#contributions)
+
+## Learning MXNet and other Educational Resources
+
+* [Gluon 60 Minute Crash Course](https://gluon-crash-course.mxnet.io/) - deep learning practitioners can learn Gluon quickly with these six 10-minute tutorials.
+* [The Straight Dope](https://gluon.mxnet.io/) - a series of notebooks designed to teach deep learning using the Gluon Python API for MXNet.
+
+
+## New MXNet APIs
+
+* [Clojure API](https://github.com/apache/incubator-mxnet/tree/master/contrib/clojure-package) - use MXNet with Clojure.
+* [Gluon Python Interface](../gluon/index.html) - train complex models imperatively and then deploy with a symbolic graph.
+* [ONNX-MXnet API](../api/python/contrib/onnx.html) - train and use Open Neural Network eXchange (ONNX) model files.
+* [Scala Infer API](../api/scala/infer.html) - model loading and inference functionality.
+
+
+## Toolkits to Extend MXNet
+
+* [Gluon CV](https://gluon-cv.mxnet.io/) - state-of-the-art deep learning algorithms in computer vision.
+* [Gluon NLP](https://gluon-nlp.mxnet.io/) - state-of-the-art deep learning models in natural language processing.
+
+
+## Model Serving
+
+* [MXNet Model Server (MMS)](https://github.com/apache/incubator-mxnet/tree/master/example/model-server/mms.md) - simple yet scalable solution for model inference.
+
+
+## Model Zoos
+
+* [Gluon Model Zoo](https://mxnet.incubator.apache.org/api/python/gluon/model_zoo.html) - models trained in Gluon and available through Gluon's model zoo API.
+* [ONNX Model Zoo](https://github.com/onnx/model-zoo) - ONNX models from a variety of ONNX-supported frameworks.
+
+
+## MXNet as a Backend
+
+* [Keras-MXNet](https://github.com/awslabs/keras-apache-mxnet) - design with Keras2 and train with MXNet as the backend for 2x or more speed improvement.
+
+
+## Contributions
+
+Do you know of a project or resource in the MXNet ecosystem that should be listed here? Or would you like to get involved by providing your own contribution? Check out the [guide for contributing to MXNet](contribute.html), and browse the [design proposals](https://cwiki.apache.org/confluence/display/MXNET/Design+Proposals) to see what others are working on. You might find something you would like to help with or use those design docs as a template for your own proposal. Use one of the [developer communication channels](https://mxnet.incubator.apache.org/community/contribute.html#mxnet-dev-communications) if you would like to know more, or [create a GitHub issue](https://github.com/apache/incubator-mxnet/issues/new) if you would like to propose something for the MXNet ecosystem.
```

Comment:
Some more things we could add to this page:

* https://github.com/NVIDIA/DALI
* https://github.com/awslabs/sockeye
* https://github.com/awslabs/mxboard

Reply:
@szha - that's a great list that chinakook has compiled. I feel like some of those items need to be added to /examples. Some could be added to ecosystem. I'm not sure if I should do that in this PR though.
It would make more sense for contributors to propose the addition of their work to the ecosystem or examples page(s), wouldn't it?

@thomelane - I've added those three in my pending update.

Reply:
@chinakook would you be ok with having your Awesome-MXNet package linked in the ecosystem page?

Reply:
@szha OK, I will add it after this PR is merged.